content/en/blog/_posts/2020-03-30-two-phased-canary-rollout-with-gloo.md

```diff
@@ -0,0 +1,772 @@
+---
+title: " Two-phased Canary Rollout with Open Source Gloo "
+date: 2020-03-30
+slug: two-phased-canary-rollout-with-gloo
+url: /blog/2020/03/Two-phased-Canary-Rollout-With-Gloo
+---
+
```

Comment:
On this blog, posts usually show the author name and optionally their organization.